README.md

```diff
@@ -11,6 +11,36 @@ Job's Done
 Job's Done is a tool heavily inspired by [Travis](https://travis-ci.org/), and works in the same way 
 in that configuring a `.jobs_done.yaml` file in your repository's root to create and trigger Continuous Integration jobs.
 
+Example of a `.jobs_done.yaml` file:
+
+```yaml
+matrix:
+  platform:
+  - "win64"
+  - "redhat64"
+
+platform-win64:build_batch_commands:
+- | 
+  python -m venv .env3   || goto eof
+  call .env3\Scripts\activate   || goto eof
+  pytest --junitxml=tests-{platform}.xml  
+ 
+platform-redhat64:build_shell_commands:
+- |
+  python3 -m venv .env3
+  source .env3/bin/activate
+  pytest --junitxml=tests-{platform}.xml  
+ 
+junit_patterns:
+- "tests.*.xml"
+```
+
+Considering this file is in repository `myproject` and was pushed into branch `feat-71`, this will generate two Jenkins jobs:

```

Comment:
I will change `in repository` for `in repository's root`.
Or did this constrain have been lifted?